analyzer/src/main/kotlin/managers/DotNet.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2019 Bosch Software Innovations
+ * Based on:
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.analyzer.managers
+
+import com.here.ort.analyzer.AbstractPackageManagerFactory
+import com.here.ort.analyzer.DotNetSupport
+import com.here.ort.analyzer.PackageManager
+import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.model.Identifier
+import com.here.ort.model.Project
+import com.here.ort.model.ProjectAnalyzerResult
+import com.here.ort.model.config.AnalyzerConfiguration
+import com.here.ort.model.config.RepositoryConfiguration
+import com.here.ort.model.xmlMapper
+import java.io.File
+
+/**
+ * Dotnet package manager
+ */
+open class DotNet(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) :
+        PackageManager(name ,analyzerConfig, repoConfig)/*, CommandLineTool */ {
+
+    class Factory : AbstractPackageManagerFactory<DotNet>("DotNet") {
+        override val globsForDefinitionFiles = listOf("dotnet.xml")
```

Comment:
So, IIUC, "dotnet.xml" is not a standard file that belongs to DotNet projects, but it is an "artificial marker" you can up with for ORT, correct? If so, I wonder why this is necessary, and why you couldn't just here list "*.csproj" instead of manually searching for such files below. Could you please elaborate?

Reply:
is now removed, was there due to misunderstanding of the ORT code. 

Also manual search for files is removed.